app/assets/javascripts/school-search/school-search.js

```diff
@@ -1,78 +1,114 @@
-import { allSchools } from './data.js'
+import { allSchools, allSchoolsWithSuccess } from './data.js';
 
-const prevOnload = window.onload
+const prevOnload = window.onload;
 window.onload = () => {
-  setupEventListeners()
-  applyFiltering()
-  prevOnload && prevOnload()
-}
+  setupEventListeners();
+  applyFiltering();
+  prevOnload && prevOnload();
+};
 
 // function that takes a list of schools and filters a list of them based on pupil premium
-function setupEventListeners () {
-  const applyFiltersButtons = document.getElementsByName('apply-school-filters')[0]
-  applyFiltersButtons.addEventListener('click', applyFiltering)
+function setupEventListeners() {
+  const applyFiltersButtons = document.getElementsByName(
+    'apply-school-filters'
+  )[0];
+  applyFiltersButtons.addEventListener('click', applyFiltering);
 }
 
-function applyFiltering () {
-  const resultsHtml = window.nunjucks.render('school-results.html', { schools: filteredSchools() })
-  document.getElementById('school-results').innerHTML = resultsHtml
+function applyFiltering() {
+  const resultsHtml = window.nunjucks.render('school-results.html', {
+    schools: filteredSchools(),
+  });
+  document.getElementById('school-results').innerHTML = resultsHtml;
 }
 
-function filteredSchools () {
-  let schools = allSchools
+function filteredSchools() {
+  let schools = getAllSchools();
```

Comment:
Probably worth comparing linter setup with Ethan, picking one, and adding a note about it in the readme. 

You could even do something like https://prettier.io/docs/en/precommit.html to include make your linter (article uses `prettier`) format everything each time you commit changes.

Reply:
Yeah, we noticed this. I'm adding eslint into the project to define shared settings